pkg/util/virt-chroot.go

```diff
@@ -0,0 +1,42 @@
+package util
```

Comment:
How about placing this in `pkg/virt-handler/virt-chroot`? It should never be used outside of `virt-handler`.

Reply:
Sure! good idea, thanks